src/main/resources/graphql/schema.graphqls

```diff
@@ -1,16 +1,96 @@
-# schema file of the microservice
+type Query {
+  """
+  Get the skill levels of the current user for all skill types (Bloom taxonomy) in a chapter of a course.
+  """
+  userSkillLevels(courseId: UUID!, chapterId: UUID!): SkillLevels!
+
+  """
+  Get the skill levels of the specified user for all skill types (Bloom taxonomy) in a chapter of a course.
+  """
+  skillLevelsForUser(userId: UUID!, courseId: UUID!, chapterId: UUID!): SkillLevels!
```

Comment:
We need those two queries also aggregated over the whole course, so userSkillLevels and skillLevelsForUser, without a chapter id. I would suggest to make the chapterId optional and note in the comment, that when the chapterId is `null`, then the skill levels for the course are returned.